changelog.md

```diff
@@ -1,3 +1,68 @@
+## Code Dx Jenkins Plugin 4.0.0 `Released 5/10/2023`
+
+### Changes
+- The "API Key" field now uses a `credential` instead of a `string` value
+- The "API Key" field ID (`key`) has been renamed to `keyCredentialId`
+- When configuring Freestyle projects, "Configure" permission on the project is required to make changes
+- When using the Pipeline Syntax Generator tool, the global "Administer" permission is required to validate the form and list projects
```

Comment:
Should this be Administrator instead of Administer?

Reply:
It should be "Administer"; "Administrator" is a role which has multiple permissions, "Administer" is a specific permission

(though this setup is weird and I agree that it should be "Administrator", but Jenkins permissions currently sets it up this way with these terms and usage)